### PR TITLE
doc: BT ISO time synch sample: align with the sample template

### PR DESCRIPTION
**Pull request by peknis**
Aligned the sample doc with the template.

### CODE REVIEW

File: samples/bluetooth/iso_time_sync/README.rst [hunk 1]
```diff
@@ -45,6 +25,26 @@ Additionally, the sample requires a connection to a computer with a serial termi
...
+The sample demonstrates the following features:
+
+ * How to send isochronous data on multiple streams so that the receiving devices and the sending device can process the data at the same time.
+ * How the receivers of the isochronous data can process the received data synchronously.
+ * How to provide the isochronous data to the controller right before it is sent on-air.
+ * How to achieve this for either broadcast (BIS) or over connections (CIS).
+
+This sample configures a single device as a transmitter of its **BUTTON1** state.
+The transmitting and receiving devices toggle **LED1** synchronously with the accuracy of a few microseconds.
+When the :kconfig:option:`LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE` Kconfig option is enabled, **LED2** is toggled upon sending or receiving data.
```
Comment (umapraseeda) [inline samples/bluetooth/iso_time_sync/README.rst:40]:
```suggestion
When the :kconfig:option:`CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE` Kconfig option is enabled, **LED2** is toggled upon sending or receiving data.
```

File: samples/bluetooth/iso_time_sync/README.rst [hunk 2]
```diff
@@ -209,7 +209,7 @@ After programming the sample to the development kits, perform the following step
    If the central is configured for transmission, it connects to multiple peripherals.
 #. On the other terminal(s), type ``p`` to configure the device(s) as connected isochronous stream peripheral(s).
 #. Select data direction.
-#. Observe that the periphrals(s) connect to the central and start receiving isochronous data.
+#. Observe that the periphral(s) connect to the central and start receiving isochronous data.
```
Comment (umapraseeda) [inline samples/bluetooth/iso_time_sync/README.rst:212]:
```suggestion
#. Observe that the peripheral(s) connect to the central and start receiving isochronous data.
```

File: samples/bluetooth/iso_time_sync/README.rst [hunk 3]
```diff
@@ -154,7 +154,7 @@ The sample code is divided into multiple source files, which makes it easier to
     The very first SDU is provided without a timestamp, because the timestamp is not known at this point in time.
   * **LED1** is configured to toggle synchronously with the **LED1** on all the receivers.
     The toggle time is determined by the TX timestamp and defined relative to the synchronization reference on the receiver.
-  * If the :kconfig:option:`LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE` Kconfig option is enabled, **LED2** is toggled right before sending the SDU.
+  * If the ``CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE`` Kconfig option is enabled, **LED2** is toggled right before sending the SDU.
```
Comment (umapraseeda) [inline samples/bluetooth/iso_time_sync/README.rst:157]: I think for all such instances, we have to use the format like :ref:`CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE <CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE>` that in turn would require a section like https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html#configuration-options where such options are defined.
```suggestion
  * If the ``CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE`` Kconfig option is enabled, **LED2** is toggled right before sending the SDU.
```

**Review by umapraseeda — CHANGES_REQUESTED** [review]

File: samples/bluetooth/iso_time_sync/README.rst [hunk 4]
```diff
@@ -21,29 +37,41 @@ The sample demonstrates the following features:
...
 
-Additionally, the sample requires a connection to a computer with a serial terminal for each development kit in use.
+.. _CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE
 
-To observe the toggling of the LEDs, use a logic analyzer or an oscilloscope.
+CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE
+   This configuration option enables immediate toggling of **LED2** when isochronous data is sent or received.
+   It allows for measurement of the mimimum end-to-end latency.
+
+.. _CONFIG_SDU_INTERVAL_US
```
Comment (umapraseeda) [inline samples/bluetooth/iso_time_sync/README.rst:64]:
```suggestion
.. _CONFIG_SDU_INTERVAL_US:
```

File: samples/bluetooth/iso_time_sync/README.rst [hunk 5]
```diff
@@ -21,29 +37,41 @@ The sample demonstrates the following features:
...
+
+CONFIG_SDU_INTERVAL_US
+   This configuration option determines how often the button value is transmitted to the receiving devices.
+
+.. _CONFIG_TIMED_LED_PRESENTATION_DELAY_US
```
Comment (umapraseeda) [inline samples/bluetooth/iso_time_sync/README.rst:69]:
```suggestion
.. _CONFIG_TIMED_LED_PRESENTATION_DELAY_US:
```

File: samples/bluetooth/iso_time_sync/README.rst [hunk 6]
```diff
@@ -21,29 +37,41 @@ The sample demonstrates the following features:
...
+
+CONFIG_TIMED_LED_PRESENTATION_DELAY_US
```
Comment (umapraseeda) [inline samples/bluetooth/iso_time_sync/README.rst:71]:
```suggestion
CONFIG_TIMED_LED_PRESENTATION_DELAY_US:
```

Comment (peknis) [reply samples/bluetooth/iso_time_sync/README.rst:71]: Suggested fix on wrong line. Line 69 needs a colon at the end.

**Review by rugeGerritsen — APPROVED** [review]

File: samples/bluetooth/iso_time_sync/README.rst [hunk 7]
```diff
@@ -21,29 +37,41 @@ The sample demonstrates the following features:
...
-You can use any combination of the development kits mentioned above, mixing different development kits.
+Check and configure the following Kconfig options:
 
-Additionally, the sample requires a connection to a computer with a serial terminal for each development kit in use.
+.. _CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE:
 
-To observe the toggling of the LEDs, use a logic analyzer or an oscilloscope.
+CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE
+   This configuration option enables immediate toggling of **LED2** when isochronous data is sent or received.
+   It allows for measurement of the mimimum end-to-end latency.
```
Comment (umapraseeda) [inline samples/bluetooth/iso_time_sync/README.rst:62]:
```suggestion
   It allows for measurement of the minimum end-to-end latency.
```

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**